hack/manifest-gen/assets/charts/eck/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Core configuration values
+operator:
+  # Name of the operator installation.
+  name: elastic-operator
+  # Namespace to install the operator to.
+  namespace: elastic-system
+  # Version of the operator. Should be set to the desired container image tag.
+  version: 1.2.0
+
+  # Labels to be added to the operator. Useful if multiple operators are to be deployed to the cluster.
+  # These labels are used by the webhook service if it is enabled as well.
+  selectorLabels:
+    control-plane: elastic-operator
+
+  # Container image options.
+  image:
+    # Full path to the operator container image excluding the tag. The tag value is derived from the operator.version field.
+    repository: docker.elastic.co/eck/eck-operator
+    # Image pull policy for the container.
+    pullPolicy: IfNotPresent
+
+  # Resource and request limits for the operator pod.
+  resources:
+    limits:
+      cpu: 1
+      memory: 150Mi
+    requests:
+      cpu: 100m
+      memory: 50Mi
+
+# Optional configuration values
+config:
+  # Set to true to enable creating cluster resources (e.g. ClusterRoles). If set to false, only namespace-level resources will be created.
+  createClusterResources: true
+
+  # Set of namespaces for the operator to manage. Set to empty array to manage all namespaces.
+  managedNamespaces: []
+
+  # Operator log level. Valid values are:
+  # -2 == ERROR
+  # -1 == WARN
+  #  0 == INFO
+  #  1 and above == DEBUG
+  logVerbosity: 0
+
+  # Endpoint for exposing Prometheus metrics. Set to 0 to disable the endpoint.
+  metricsPort: 0
+
+  # Default container registry to be used by the operator when pulling Elastic Stack container images.
+  containerRegistry: docker.elastic.co
+
+  # Parallelism of the operator.
+  maxConcurrentReconciles: 3
+
+  # Experimental APM tracing configuration for the operator.
+  tracing:
+    enabled: false
+    config:
+      ELASTIC_APM_SERVER_URL: http://localhost:8200
+      ELASTIC_APM_SERVER_TIMEOUT: 30s
```

Comment:
would be nice to also include/document `ELASTIC_APM_SERVER_TOKEN` here. Maybe keep it commented?

```yaml
      ELASTIC_APM_SERVER_URL: http://localhost:8200
      ELASTIC_APM_SERVER_TIMEOUT: 30s
      # ELASTIC_APM_SERVER_TOKEN: <token>
```